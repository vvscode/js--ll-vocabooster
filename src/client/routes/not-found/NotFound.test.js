import React from 'react';
import { shallow } from 'enzyme';
import Component from './NotFound.component';

describe('NotFound', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {};

    getShallowComponent = () => shallow(<Component {...props} />);
  });

  describe('render', () => {
    it('should render with title', () => {
      props.title = 'Some title';
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
