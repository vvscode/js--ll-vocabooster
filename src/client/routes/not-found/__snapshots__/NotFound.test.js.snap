// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound render should render with title 1`] = `
<div
  className="NotFound__root"
>
  <div
    className="NotFound__container"
  >
    <h1>
      Some title
    </h1>
    <div
      className="NotFound__mainWrapper"
    >
      <div
        className="NotFound__errorWrapper"
      >
        <div>
          <img
            alt=""
            src="404.svg"
          />
        </div>
        <div>
          <div>
            <h2
              className="NotFound__title"
            >
              Page not found
            </h2>
          </div>
          <div
            className="NotFound__errorCode"
          >
            Error code: 404
          </div>
          <div>
            <Link
              className="NotFound__button"
              replace={false}
              to="/health-center"
            >
              Back to home
            </Link>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
